02_TrackWeb/src/stats.ts

```
@@ -0,0 +1,221 @@
+                    console.log('URL copied to clipboard');
+                } catch (err) {
+                    console.error('Failed to copy URL: ', err);
+                    showError('Failed to copy URL.');
+                }
+            });
+        }
+    });
+    
+    detailsCell.appendChild(innerTable);
+    detailsRow.appendChild(detailsCell);
+    domainRow.parentNode!.insertBefore(detailsRow, domainRow.nextSibling);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)